packages/psss-server/src/types.ts

```diff
@@ -36,7 +36,7 @@ export interface Credentials {
 
 export type QueryParameters = Record<string, string>;
 
-export type RequestBody = Record<string, unknown>;
+export type RequestBody = Record<string, unknown> | Record<string, unknown>[];
```

Comment:
The body of the `/changes` endpoint in `meditrak-server` is an array of JSON objects

Reply:
We shouldn't be using the `/changes` endpoint in `meditrak-server` - that's intended for sync. Could you add that to the tech debt card too?